config/rbac/riff-clusterrole.yaml

```diff
@@ -8,10 +8,10 @@ rules:
     verbs: ["get", "list", "watch", "create", "delete", "update"]
   - apiGroups: ["apiextensions.k8s.io"]
```

Comment:
I'm not even sure we need anything on CR *definitions* do we? We don't handle those programmatically; only instances of the definitions.